pkg/ebpf/events_pipeline.go

```diff
@@ -268,9 +268,9 @@ func (t *Tracee) decodeEvents(ctx context.Context, sourceChan chan []byte) (<-ch
 			// thus the need to continue with those within the pipeline.
 			if t.matchPolicies(evt) == 0 {
 				_, hasDerivation := t.eventDerivations[eventId]
-				_, hasSignature := t.eventSignatures[eventId]
+				isSignature := t.policyManager.IsSignature(eventId)
```

Comment:
I don't think this change is correct.
We want to check if the event have signatures that depend on it (`hasSignature`) and not if the event itself is a signature (`isSignature`)

Reply:
I think I've changed the nomenclature but need to double check the logic. Soon I'll be returning to this. Tks.

Reply:
Well, you are right. But I detected the main issue. As I only moved logic into the new place, I noticed that it didn't save the events in the map as required by sigs, it just set if it's a signature. So, at that stage it didn't work as a shortcut.

My last change includes:

```
  Fix t.eventSignatures[eventId] logic, now accessible via policy
  manager IsRequiredBySignature(), since the former was only flagging
  the event as signature, but not as required by one.
```

Reply:
E2E 1783: ✅

Reply:
@geyslan, I tested it and compared it with the main branch (which, as you mentioned, has a bug).